front/components/assistant/GalleryAssistantPreviewContainer.tsx

```diff
@@ -48,21 +51,41 @@ const useAssistantUpdate = (
         : `/api/w/${owner.sId}/members/me/agent_list_status`;
     const method = flow === "workspace" ? "PATCH" : "POST";
 
+    const detailedAssitantRes = await fetch(
```

Comment:
This one isn't super great but we need to pass the whole thing to POST/PATCH

Reply:
Yeah let's create a new route for that.
To keep things as general as possible we could make a route that lets you update the Light version of the agent ?

Reply:
Yes that sounds like a good solution

Reply:
(still fixing this one but rest is g2g)

Reply:
Ok should be good now